model/README.md

```diff
@@ -0,0 +1,9 @@
+The model is based on https://github.com/ZackAkil/deep-tic-tac-toe.
```

Comment:
I it may be worth dropping a line here about tfserve so people understand that architecture and what the `tf` means in the code.

Reply:
Hm, not exactly sure if this is the right place to explain tf :thinking: 
The blog post is supposed to explain all the details so I guess we can drop a link into this or the top-level README to it once published.

Reply:
Not a full explanation but no game devs will know what tf is and will be confused about this technology that is listening to another port o nthe local server.

Reply:
I updated the top-level README and removed the exposed port, it's not actually needed.
This particular README was intended more as  a cheat sheet for whoever wanted to mess with the model, i.e. the basic knowledge of tf is assumed so I'd leave it as is.